simulator/travis/prepare_simulation.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ev
+
+lein deps
+
+cd ../scheduler
+lein deps
+lein uberjar
+
+cd ../simulator/travis
+unzip datomic-free-0.9.5394.zip
```

Comment:
This doesn't include the metatransaction jar, but the jar is added to a spot which will be included in Datomic's classpath on the next line.